website/src/plugins/github-release-plugin.ts

```diff
@@ -0,0 +1,178 @@
+import type { Plugin } from '@docusaurus/types';
+import { Octokit } from '@octokit/rest';
+
+interface DownloadData {
+  version: string;
+}
+
+// --- Data type definitions for each OS ---
+
+export interface LinuxDownloadData extends DownloadData {
+  flatpak: string;
+  amd64: string;
+  arm64: string;
+}
+
+export interface MacosDownloadData extends DownloadData {
+  universal: string;
+  x64: string;
+  arm64: string;
+  airgapsetupX64: string;
+  airgapsetupArm64: string;
+}
+
+export interface WindowsDownloadData extends DownloadData {
+  binaryX64: string;
+  binaryArm64: string;
+  setupX64: string;
+  setupArm64: string;
+  airgapsetupX64: string;
+  airgapsetupArm64: string;
+}
+
+// --- Combined data types ---
```

Comment:
either keep with something helpful or drop the comment (and drop this style with --- )

Reply:
87bc4220e48345790ff376b31a7af30f9f2f089e